coalib/settings/Setting.py

```diff
@@ -13,6 +14,22 @@ def path_list(obj, *args, **kwargs):
     return obj.__path_list__(*args, **kwargs)
 
 
+def glob(obj, *args, **kwargs):
+    """
+    Creates a path in which all special glob characters in all the
+    parent directories in the given setting are properly escaped.
+    """
```

Comment:
Honestly, I don't know where we can use `*args and **kwargs`. @sils1297 had added it in https://github.com/coala-analyzer/coala/commit/74551ef9f707acb74b0b3cab190b68a53639ee70#diff-2663e4c825081e483f6ed8d673b78545 so, I wanted to have the same parameters for `glob` and `glob_list` since they are very similar in function to `path` and `path_list`.

But I'll add the rest of the stuff right away.